### PR TITLE
[WIP] Marketing

### PR DESCRIPTION
**Pull request by Butt4cak3**
The README is lacking buzzwords. We need more of those to appeal to people who have no idea how software works. This PR is work-in-progress, because I want everyone to suggest some cool words that we could add to make the README even more **awesome**. Speaking of awesome - that's probably a word that should appear in the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
...
 Now with docker support for people who want to make box in box!
+
+---
+
+Box your boxes like never before! box is **blazing fast** and **easy to use**. You won't have to waste any more time and money with complicated boxes. box is modular, so it's **easy to integrate** into your stack out of the box.
+
+box is proven to outperform its competitors by over 20%! Just look at these numbers:
+
+| box | 126% |
+| its competitors | 99% |
```
Comment (SjoerdNijboer) [inline README.md]: Needs to mention blockchain and AI.
Also, we're doing zero overhead boxing.

**Comment by Butt4cak3** [comment]
I messed up the table markdown. I'm already working on it.

**Comment by june128** [comment]
We should also mention that we utilize Docker and CI in our development, so users immediately know that box is quality software!

**Comment by SjoerdNijboer** [comment]
Hmm, you're right.
Might as well mention we're using a miltithreaded buildscript

**Comment by SjoerdNijboer** [comment]
I just realized. We're using gcc and clang for CI, but box doesn't officially support clang.
So we might as well mention that explicitly to make us look more professional.

**Comment by Butt4cak3** [comment]
I incorporated some more buzzwords like Blockchain, AI, Docker, Cloud and Big Data. I think that's enough for now.

**Pull request merged**